app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Show users pages</h1>
\ No newline at end of file
+<h1>User ID: <%= @user.id %> </h1>
+<h1>Name: <%= @user.name %> </h1>
```

Comment:
- [ ] Every page suppose to have only one `<h1>` tag, Please change the second `<h1>` tag for semantic reasons.

Reply:
@Omar-Muhamad Thank you for reviewing my project, I will make the necessary corrections as suggested